config/config.go

```diff
@@ -187,14 +187,15 @@ type GrafanaVariablesConfig struct {
 
 // TracingConfig describes configuration used for tracing links
 type TracingConfig struct {
-	Auth                 Auth     `yaml:"auth"`
-	Enabled              bool     `yaml:"enabled"` // Enable Jaeger in Kiali
-	InClusterURL         string   `yaml:"in_cluster_url"`
-	IsCore               bool     `yaml:"is_core,omitempty"`
-	NamespaceSelector    bool     `yaml:"namespace_selector"`
-	URL                  string   `yaml:"url"`
-	UseGRPC              bool     `yaml:"use_grpc"`
-	WhiteListIstioSystem []string `yaml:"whitelist_istio_system"`
+	Auth                 Auth              `yaml:"auth"`
+	Enabled              bool              `yaml:"enabled"` // Enable Jaeger in Kiali
+	InClusterURL         string            `yaml:"in_cluster_url"`
+	IsCore               bool              `yaml:"is_core,omitempty"`
+	NamespaceSelector    bool              `yaml:"namespace_selector"`
+	QueryScope           map[string]string `yaml:"query_scope,omitempty"`
```

Comment:
"QueryScope" sounds strange to me in this context.
I think it can create confusion, but something like "AdditionalTags" would be clear for maintaining this in the future.

Reply:
I understand what you are saying but I like the synergy between the naming for both this and the Prometheus external service.  The use case being solved is the same.  If we ever had to, we could also add `AdditionalTags` and it would simply do the same thing.  But for the user trying to solve the problem of unified storage, I think it's good to have the same naming here and for prom.